public/docs/_examples/dependency-injection/dart/lib/app_component.dart

```diff
@@ -11,6 +11,7 @@ import 'user_service.dart';
 //PENDING: check whether we intend to hide injector_component.dart & providers_component.dart; if so, change docregion name?
```

Comment:
Should this comment really be visible? Also, it doesn't reflect test_component.dart.